experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { TypedEventEmitter } from "@fluidframework/common-utils";
+import { Container } from "@fluidframework/container-loader";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import { IEvent } from "@fluidframework/common-definitions";
+import { LoadableObjectClass } from "./types";
+import { RootDataObject } from "./rootDataObject";
+
+interface IFluidContainerEvents extends IEvent {
+    (event: "connected", listener: (clientId: string) => void): void;
+    (event: "dispose" | "disconnected", listener: () => void): void;
+}
+
+interface IFluidContainer {
+    close: Container["close"];
+    closed: Container["closed"];
+    connected: Container["connected"];
```

Comment:
I've mentioned this before, but please don't pull signatures from the `Container`.  Even if they happen to be an exact match the point of `IFluidContainer` is to re-establish an API surface that is better controlled.

As an example of why this is bad, `close()` takes an `error?: ICriticalContainerError` argument on `Container`, which is not something we intend to provide at this layer.